augly/image/functional.py

```diff
@@ -1073,10 +1080,21 @@ def overlay_emoji(
         specify `bbox_format` if `bboxes` is provided. Supported bbox_format values are
         "pascal_voc", "pascal_voc_norm", "coco", and "yolo"
 
+    @param seed: seed for numpy random generator to select random pixels for shuffling
+
     @returns: the augmented PIL Image
     """
     image = imutils.validate_and_load_image(image)
 
+    np.random.seed(seed)
+
+    if isinstance(emoji_size, tuple):
+        emoji_size = np.random.uniform(emoji_size[0], emoji_size[1])
```

Comment:
```suggestion
        assert emoji_size[0] < emoji_size[1], (
            "emoji_size must be a float or a tuple [low, high) to sample the value from"
        )
        emoji_size = np.random.uniform(emoji_size[0], emoji_size[1])
```
Give a useful error message if a bad tuple is given